Healthy/Classes/Modules/Splash/DemoViewController.swift

```diff
@@ -0,0 +1,225 @@
+//
+//  OnboardingViewController.swift
+//  Healthy
+//
+//  Created by dodo on 15/05/2023.
+//
+
+
```

Comment:
```suggestion
//
//  OnboardingViewController.swift
//  Healthy
//
//  Created by dodo on 15/05/2023.
//
```

Headers should be removed.

Reply:
👀